src/Microsoft.Identity.Client/Internal/PoP/IPoPCryptoProvider.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.Internal.PoP
+{
+    public interface IPoPCryptoProvider
```

Comment:
I did not want to take a dependency on Wilson just yet, so I am separating Wilson behind this interface.